### PR TITLE
pokemon images

### PR DESCRIPTION
**Pull request by xBunnicornx**
background + images + banner + shadow

### CODE REVIEW

**Comment by roman-yakobnyuk** [comment]
@xBunnicornx looks like your build has failed, please check it locally.

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,22 +1,54 @@
...
+              alt="running--v1"
+            />{" "}
+            Run Away
+          </button>
+        </div>
+
         <Image
-          src="/images/pokemon/001.png"
+          src={`/images/backgrounds/background_${randomBackgroundNumber}.jpg`}
           alt="Next.js Logo"
```
Comment (roman-yakobnyuk) [inline src/app/page.tsx:31]: Please replace this with correct alternative text.

File: src/app/styles/page.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  width: auto;
+  height: 50px;
+  background: #000000;
+  position: relative;
+}
+.banner:after {
+  position: relative;
+  height: 0;
+  width: 5px;
+  right: -245px;
```
Comment (roman-yakobnyuk) [inline src/app/styles/page.css:31]: There is some weird CSS here, I do not really understand it.

File: src/app/styles/page.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  bottom: 10px;
+  border-top: 10px solid #000000;
+  border-right: 10px solid transparent;
+}
+.tittle {
```
Comment (roman-yakobnyuk) [inline src/app/styles/page.css:36]: There is a typo here.

File: src/app/styles/page.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+}
+.background {
+  display: flex;
+  flex-direction: row;
+  max-width: 900px;
+  position: absolute;
+  top: 5%;
+  left: 13%;
+}
+@media only screen and (max-width: 480px) {
```
Comment (roman-yakobnyuk) [inline src/app/styles/page.css:49]: Please make sure to leave an empty line between blocks.

**Review by roman-yakobnyuk — CHANGES_REQUESTED** [review]

File: src/app/page.tsx [hunk 5]
```diff
@@ -1,22 +1,39 @@
...
-        <p>Homepage</p>
-      </div>
-
-      <div className={styles.center}>
+  
+    <div className="main"><div className="banner">  <div className="tittle">Who is that Pokemon?</div></div>
+    
+        <div className="background">
+          <div className="run"> 
+          <button><img  width="20" height="20" src="https://img.icons8.com/ios/50/running--v1.png" alt="running--v1"/>  Run Away</button>
```
[the remark this answers is not in the document]
Comment (xBunnicornx) [reply src/app/page.tsx]: yeah i fixed it.

File: src/app/styles/page.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  width: 500px;
+  height: 100px;
+  left: 45%;
+  text-align: center;
+  background: transparent;
+  position: absolute;
+  top: 20px;
+  z-index: 3;
+  border-left-color: transparent white;
+  clip-path: polygon(45px 0, 100% 0, 100% 100%, 0 100%);
```
[the remark this answers is not in the document]
Comment (xBunnicornx) [reply src/app/styles/page.css:12]: this is for the banner.

File: src/app/styles/page.css [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+  padding: 8px;
+  font-size: 20px;
+  box-shadow: 0 3px rgba(0, 0, 0, 0.233);
+  position: relative;
+  top: 70px;
+  left: 220px;
+  width: 180px;
+}
+.run img {
+  -moz-transform: scaleX(-1);
```
[the remark this answers is not in the document]
Comment (xBunnicornx) [reply src/app/styles/page.css:73]: cause the run icon. it face opposite site. so i tried to flip it

**Pull request merged**